composer.json

```diff
@@ -47,8 +47,9 @@
         "twig/twig": "^3.0"
     },
     "require-dev": {
+        "ext-pdo_sqlite": "*",
```

Comment:
why is it now needed ?

Reply:
Well running tests without it fails so I thought let's make it explicit?